Document the transition to Venafi Cloud OutagePREDICT (#503)
cert-manager/website: content/en/docs/configuration/venafi.md

### Creating a Venafi Cloud Issuer

> ⚠ Since cert-manager v1.3 you will need to update your Venafi Cloud configuration to use OutagePREDICT instead of DevOpsACCELERATE.

Reviewer: Since implies something that already happened, from is present/future (in my opinion!)

```suggestion
⚠ From cert-manager v1.3 you will need to update your Venafi Cloud configuration to use OutagePREDICT instead of DevOpsACCELERATE.
```

Author: Done. Although this is merging into release-next branch which will only be shown when browsing the 1.3 docs.

With this update, the zone format changes from a UUID to a string of the form `<Application Name>\<Issuing Template Alias>`.

> Please read [cert-manager 1.2 to 1.3 upgrade notes][] and [Venafi Cloud Prerequisites][] for for further information.

Author: ```suggestion
Please read [cert-manager 1.2 to 1.3 upgrade notes][] and [Venafi Cloud Prerequisites][] for further information.
```

> [cert-manager 1.2 to 1.3 upgrade notes]: ../installation/upgrading/upgrading-1.2-1.3/

Author: Fix link

> ⚠ From cert-manager `v1.3` you will need to update your Venafi Cloud configuration to use `OutagePREDICT` instead of `DevOpsACCELERATE`.
> With this update, the zone format changes from a UUID to a string of the form `<Application Name>\<Issuing Template Alias>`.
> Please read [cert-manager 1.2 to 1.3 upgrade notes][] and [Venafi Cloud Prerequisites][] for further information.
> 
> [cert-manager 1.2 to 1.3 upgrade notes]: ../../installation/upgrading/upgrading-1.2-1.3/
> [Venafi Cloud Prerequisites]: https://github.com/Venafi/vcert/blob/v4.13.1/README-CLI-CLOUD.md#prerequisites

Reviewer 2: ```suggestion
{{% pageinfo color="warning" %}}

⚠ From cert-manager `v1.3` onwards, you will need to update your Venafi Cloud
configuration to use `OutagePREDICT` instead of `DevOpsACCELERATE`. With this
update, the zone format changes from a UUID to a string of the form:

```text
    <Application Name>\<Issuing Template Alias>
```

Please read [cert-manager 1.2 to 1.3 upgrade notes][] and [Venafi Cloud
Prerequisites][] for further information.

[cert-manager 1.2 to 1.3 upgrade notes]: ../../installation/upgrading/upgrading-1.2-1.3/
[venafi cloud prerequisites]: https://github.com/Venafi/vcert/blob/v4.13.1/README-CLI-CLOUD.md#prerequisites

{{% /pageinfo %}}
```

It adds a nice red square to outline the warning:

![Screenshot 2021-04-07 at 17 22 47](https://user-images.githubusercontent.com/2195781/113892092-fa61d880-97c5-11eb-9255-1e69241036d2.png)
